<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinGame</class>
 <widget class="QDialog" name="JoinGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>718</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>718</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JoinGame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/assets/images/image_4.png);</string>
  </property>
  <widget class="QComboBox" name="matchesBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: transparent;
border-image: transparent;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>126</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: transparent;
border-image: transparent;
font: 75 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(69, 69, 69);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Select a match&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 13pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(77, 77, 77);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(77, 77, 77);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="player2Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>126</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(69, 69, 69);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Local player 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(77, 77, 77);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(77, 77, 77);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Refrash</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomGameButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(77, 77, 77);
background-image: transparent;
border-image: transparent;</string>
   </property>
   <property name="text">
    <string>Random Game</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
